File: POO/Ejercicio 3/PythonVersion/com/Numero.py
class Numero:
    # Atributos
    valor = int
    
    # Constructor
    def __init__(self, pNum):
        self.valor = pNum
    
    # Métodos
    def get_valor(self) -> int:
        return self.valor
    def set_valor(self, pNum):
        self.valor = pNum
    
    def to_string(self) -> str:
        text = str
        text += str(self.get_valor())
        return text

File: POO/excepciones/JavaVersion/src/com/Numero.java
package com;

public class Numero {
    // Atributos
    private double valor = 0;

    // Constructor
    public Numero(double pNum){
        this.valor = pNum;
    }

    // Methods
    public double getValor() {
        return valor;
    }
    public void setValor(double valor) {
        this.valor = valor;
    }
    
    @Override
    public String toString() {
        String text = "";
        text += this.getValor();
        return text;
    }
}
